standards/README.md

```diff
@@ -2,6 +2,18 @@
 
 This folder contains all current standards for software development.
 
+Our standards are intended to:
+
+- Make the produced code more easily read and understood
+- Make the code more easily maintainable
+- Make the code more reliable
+
+## Versioning
+
+Our standards are updated on an ongoing basis. Anything using these standards should therefore refer to these standards not only be name, for example **C# coding standards**, but also by version, for example **1.0**.
```

Comment:
As we are using a version control system for these standards, I don't know that we need to explicitly number them as well. I'm happy to refer to version numbers of external standards where they exist, but it doesn't seem worth the overhead internally.
I think this standard should say to refer to the date that the applicable standards were adopted, so it would be straightforward to see what versions were in effect at that point.

Reply:
💯